magenta/music/notebook_utils.py

```diff
@@ -16,6 +16,11 @@
 # internal imports
 import IPython
 
+import collections
+import bokeh
```

Comment:
We should update magenta/tools/pip/setup.py to have dependencies on pandas and bokeh.

Should also add a dependency in the BUILD file with "# bokeh dep" and "# pandas dep" that we can handle internally.